source/cli.js

```diff
@@ -29,6 +32,7 @@ const cli = meow(`
 	  --no-yarn           Don't use Yarn
 	  --contents          Subdirectory to publish
 	  --no-release-draft  Skips opening a GitHub release draft
+	  --auto              Categorize changes and determine version automatically
```

Comment:
I think that `--interactive` might be a more fitting name for this feature. However, one issue that might arise is that we currently call the current version selection UI "interactive UI", which might cause some confusion.

Reply:
`--interactive` for this would be confusion, I think.

Reply:
It reminds me about `git rebase master --interactive`. If users can edit in another clean screen, that will be much more quick than selecting one by one.

Reply:
@chinesedfan Cool idea 👍🏻 After this is merged, we can create an issue for it, and implement it sometime in the future.